src/GitVersionTfsTask/GitVersion.ps1

```diff
@@ -54,6 +54,13 @@ if($additionalArguments)
   $argsGitVersion = $argsGitVersion + " $additionalArguments"
 }
 
+if(!$additionalArguments -or !$additionalArguments.Contains("-debug"))
+{
+  if([string]::IsNullOrEmpty($env:SYSTEM_DEBUG) -or $env:SYSTEM_DEBUG -ne "true"){
```

Comment:
I removed this for the moment. I figured it can be another PR which we can look at separately